index.html

```diff
@@ -414,7 +438,10 @@ <h3>MathML Element Mappings</h3>
 		  </td>
 		  <td class="axapi">
 		    <span class="role">AXRole: <code>NSAccessibilityGroupRole</code></span><br />
-		    <span class="subrole">AXSubrole: <code>AXMathSquareRoot</code></span>
+		    <span class="subrole">AXSubrole: <code>AXMathSquareRoot</code></span><br />
+		    <span class="attrs">AXAttributes:
+		      <code>AXMathRootRadicand</code> pointing to a first in-flow child
```

Comment:
As I said, the structure is `<msqrt>child1 child2 child3 ... childN</msqrt>`

Returning first child is indeed what WebKit currently does. However, I wonder whether this is actually broken, AccessibilityMathMLElement::mathRadicandObject refers to https://bugs.webkit.org/show_bug.cgi?id=146452

Maybe AXMathRootRadicand should instead return an array of children? Probably worth asking jcraig@apple.com and cfleizach@apple.com to see what is most convenient for VoiceOver.

Reply:
> As I said, the structure is `<msqrt>child1 child2 child3 ... childN</msqrt>`

I should have read the spec more carefully, instead of following implementations blindly.

> Returning first child is indeed what WebKit currently does. However, I wonder whether this is actually broken, AccessibilityMathMLElement::mathRadicandObject refers to https://bugs.webkit.org/show_bug.cgi?id=146452
> 
> Maybe AXMathRootRadicand should instead return an array of children? Probably worth asking [jcraig@apple.com](mailto:jcraig@apple.com) and [cfleizach@apple.com](mailto:cfleizach@apple.com) to see what is most convenient for VoiceOver.

Agreed. I'd say since WebKit returns an array (currently containing only one element) then it looks like they are all set with the expected format, and all we need to do is to update the spec to indicate multiple elements may be returned.

CC'ing @cookiecrook for sanity check

Reply:
> > As I said, the structure is `<msqrt>child1 child2 child3 ... childN</msqrt>`
> 
> I should have read the spec more carefully, instead of following implementations blindly.
> 
> > Returning first child is indeed what WebKit currently does. However, I wonder whether this is actually broken, AccessibilityMathMLElement::mathRadicandObject refers to https://bugs.webkit.org/show_bug.cgi?id=146452
> > Maybe AXMathRootRadicand should instead return an array of children? Probably worth asking [jcraig@apple.com](mailto:jcraig@apple.com) and [cfleizach@apple.com](mailto:cfleizach@apple.com) to see what is most convenient for VoiceOver.
> 
> Agreed. I'd say since WebKit returns an array (currently containing only one element) then it looks like they are all set with the expected format, and all we need to do is to update the spec to indicate multiple elements may be returned.
> 
> CC'ing @cookiecrook for sanity check

I confused msqrt and mmultiscripts elements ^. Indeed in case of msqrt it looks like WebKit implementation should be adjusted. I'll file a followup a bug on it I guess to keep things clean.